### PR TITLE
HOMS-207 Permit all parameters by default

### PR DESCRIPTION
**Pull request by TimAle**
* Fix BP form submit + test
* Set UTC±00 timezone for tests

### CODE REVIEW

**Comment by TimAle** [comment]
UPD: fake response from Activity fixed.

File: hbw/app/controllers/hbw/tasks_controller.rb [hunk 1]
```diff
@@ -66,7 +66,7 @@ def task_id
     end
 
     def form_data
-      params[:form_data]
+      params[:form_data].to_h
```
Comment (sugarfree1) [inline hbw/app/controllers/hbw/tasks_controller.rb]: ```to_h``` is not needed though. Check it, please.

File: app/views/orders/_filter_by.html.haml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-- custom_fields = custom_fields_for_filter(params[:order_type_id], params[:custom_fields].permit!.to_h) if params[:custom_fields]
+- custom_fields = custom_fields_for_filter(params[:order_type_id], params[:custom_fields].to_h) if params[:custom_fields]
```
Comment (sugarfree1) [inline app/views/orders/_filter_by.html.haml]: ```.to_h```

File: hbw/app/controllers/hbw/tasks_controller.rb (hunk 1 above)
Comment (TimAle) [reply hbw/app/controllers/hbw/tasks_controller.rb]: Checked. `ActionController::Parameters` object has no `map` method.

**Comment by TimAle** [comment]
UPD: Set UTC±00 timezone for tests.

**Pull request merged**